examples/search.js

```diff
@@ -17,7 +16,7 @@ goog.require('ol.source.Vector');
 
 
 /** @type {!angular.Module} **/
-app.module = angular.module('app', ['ngeo']);
+app.module = angular.module('app', [ngeo.module.name]);
```

Comment:
The example depended on the main ngeo module. The search module is already a dependency of the ngeo module so there is no need to add it here.